taskcluster/focusios_taskgraph/job.py

```diff
@@ -0,0 +1,130 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is a copy of https://github.com/mozilla-mobile/firefox-ios/blob/main/taskcluster/ffios_taskgraph/job.py